Dockerfile-azure-pipelines

```diff
@@ -0,0 +1,26 @@
+# gcr.io/distroless/python3-debian10 (runtime env is using 3.7 and that's imporatant for native dependencies)
+FROM python:3.7-slim AS builder
+
+WORKDIR /
+
+ENV PYTHONUNBUFFERED=1
+
+# Poetry setup
+RUN pip3 install poetry
+RUN poetry config virtualenvs.create false
+
+COPY poetry.lock .
+COPY pyproject.toml .
+
+RUN poetry config virtualenvs.create false
+RUN poetry export -f requirements.txt --output requirements.txt
+RUN pip3 install --target=/app -r requirements.txt --no-deps
+
+# Keep the same folder structure for imports
+COPY cognite/transformations_cli/ /app/cognite/transformations_cli/
+
+# Distroless does not work with Azure pipelines
+FROM python:3.7-buster
+COPY --from=builder /app /app
+ENV PYTHONPATH /app
+
```

Comment:
Is the entry point missing or not needed? 🤔 
```
ENTRYPOINT [ "python", "/app/cognite/transformations_cli/__main__.py" ]
```

Reply:
thanks, i'll approve

Reply:
This Dockerfile is very similar to what we have for Azure Pipelines with the function deploy action:
https://github.com/cognitedata/function-action-oidc/blob/master/Dockerfile-azure-pipelines